changelogs/fragments/322-add-l2tp-client-interface-configuration.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - api_info, api_modify, _api_data.py - add support l2tp-client configuration (https://github.com/ansible-collections/community.routeros/pull/322).
```

We usually only mention the user-facing modules:
```suggestion
  - api_info, api_modify - add support ``interface l2tp-client`` configuration (https://github.com/ansible-collections/community.routeros/pull/322).
```